plugin-desktopswitch/desktopswitch.h

```diff
@@ -80,6 +80,7 @@ class DesktopSwitch : public QObject, public ILxQtPanelPlugin
     LxQt::GridLayout *mLayout;
     int mRows;
     QScopedPointer<NETRootInfo> mDesktops;
+    int mLabelType;
```

Comment:
Yes, sure, is it okay if it's content is statically defined, thus not actually linke to the content of the combo box. Tying up a combobox and an enum seems unnecessarly complicated.